activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -112,7 +112,7 @@ def indexes(table_name) # :nodoc:
 
             orders = {}
             opclasses = {}
-            include_columns = include ? include.split(",").map(&:strip) : []
+            include_columns = include ? include.split(",").map { |c| c.strip.delete('"') } : []
```

Comment:
There is already a helper to remove the surrounding quote.

I also added something to convert the double-double-quotes.

```suggestion
            include_columns = include ? include.split(",").map {|c| Utils.unquote_identifier(c.strip.gsub('""', '"'))} : []
```

Reply:
Thanks 👍, updated! I hope people do not use (at least explicitly) quotes in their column or table names 😆